helmfile/values/ingress-nginx.yaml.gotmpl

```diff
@@ -83,43 +103,42 @@ controller:
     enabled: false
     {{ end }}
 
+  metrics:
+    port: 10254
+    # if this port is changed, change healthz-port: in extraArgs: accordingly
```

Comment:
Same here
```suggestion
```

Reply:
I usually like to keep this for easy access, but ok I will delete it.